settings/settings.go

```diff
@@ -29,13 +34,14 @@ func InitSettings() {
 	SMTPPassword = os.Getenv("SMTP_PASSWORD")
 
 	FromEmail = os.Getenv("FROM_EMAIL")
+	return nil
 }
 
 var EMAIL_INTERVAL time.Duration = 24 * time.Hour
 
 var Debug bool = false
 
-// Webserver configuration
+// Web server configuration
 var Port string = "8080"
```

Comment:
you have a lot of setting variables, why don't put them in one struct? Like `Config` or so, which will contain all configuration values

Reply:
Really liked your approach to it, so adapted it for my codebase!